test/Microsoft.Sbom.Api.Tests/Config/Validators/ManifestInfoValidatorTests.cs

```diff
@@ -15,13 +15,17 @@ public class ManifestInfoValidatorTests
 {
     private readonly Mock<IAssemblyConfig> mockAssemblyConfig = new Mock<IAssemblyConfig>();
 
+    [DataRow("randomName", "2.2")]
+    [DataRow("randomName", "3.0")]
+    [DataRow("SPDX", "randomVersion")]
```

Comment:
"SPDX" is (I think) case-sensitive. Should we include at least 1 case-variant of "SPDX"?